### ISSUE TITLE
Distinguishing direct messages in notification

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#63]
```
What steps will reproduce the problem?
1. Receive direct message
2. Look for it on the timeline
3. Swear to God and switch to the Direct Messages

What is the expected output? What do you see instead?
Expected: Some kind of label or tag appearing next to the message in
notification
Got: Simply message's text

What version of the product are you using? On what operating system?
Pino 0.1.4 from your ppa on Ubuntu Karmic
```

Original issue reported on code.google.com by `rebarned@gmail.com` on 1 Feb 2010 at 7:45

**Comment by GoogleCodeExporter** [comment]
```
Weird thing, it should be low-priority enhancement proposal, not a defect. 
```

Original comment by `rebarned@gmail.com` on 1 Feb 2010 at 7:48

**Comment by GoogleCodeExporter** [comment]
```
You want to have this labels or tags in notifications bubbles? Special fo DM? I 
don't 
realy understand.
```

Original comment by `tro...@gmail.com` on 1 Feb 2010 at 8:30

**Comment by GoogleCodeExporter** [comment]
```
Yes, like, "*dm icon or simply DM:* message text" instead of "message text".
```

Original comment by `rebarned@gmail.com` on 1 Feb 2010 at 8:38

**Comment by GoogleCodeExporter** [comment]
```
Ok, now I understand :)
```

Original comment by `tro...@gmail.com` on 1 Feb 2010 at 8:43
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Great :)
Btw, aren't you from Russia? I saw a guy with the same nick on linux.org.ru

```

Original comment by `rebarned@gmail.com` on 1 Feb 2010 at 8:50

**Comment by GoogleCodeExporter** [comment]
```
I'm from Ukraine.
```

Original comment by `tro...@gmail.com` on 1 Feb 2010 at 9:10

**Comment by GoogleCodeExporter** [comment]
```
Sorry for that, no offence intended.
So you're probably that guy.
Thanks and good luck with this, hope you won't lose your interest too quick.
```

Original comment by `rebarned@gmail.com` on 1 Feb 2010 at 9:19

**Comment by GoogleCodeExporter** [comment]
```
Closed by 3845d46616f3 revision.
```

Original comment by `tro...@gmail.com` on 5 Feb 2010 at 1:00
- Changed state: **Fixed**
